tests/unittests/Foundation/ReferenceFoundation/TestNSURL.mm

```diff
@@ -73,25 +76,42 @@
     id testRoot =
         [NSPropertyListSerialization propertyListWithData:data options:static_cast<NSPropertyListReadOptions>(0) format:nil error:&error];
     if (!testRoot || error) {
+#ifdef _WIN32
         TraceError(L"TestNSURL", L"Unable to deserialize property list data");
+#else
```

Comment:
Would you "Interop" these with the makefile? I'd rather not add ifdef's to Traces and Logs?